fabtests/benchmarks/benchmark_shared.c

```diff
@@ -181,6 +188,12 @@ int bandwidth(void)
 
 	if (opts.dst_addr) {
 		for (i = j = 0; i < opts.iterations + opts.warmup_iterations; i++) {
+			if (ft_check_opts(FT_OPT_VERIFY_DATA | FT_OPT_ACTIVE)) {
+				ret = ft_fill_buf((char *) tx_buf + ft_tx_prefix_size(), opts.transfer_size);
+				if (ret)
+				return ret;
```

Comment:
It seems the indent is not correct in L194.

Reply:
Thank you! fixed.